### DISCUSSION TITLE
Installing PETSC with MPICH 3.3 and GCC9

### DISCUSSION THREAD

**Discussion by jinca** [idaholab/moose#16542] [Q&A Getting Started] [answered]
Hello, these are the modules I am using:

  1) slurm                 3) turbovnc/2.0.1        5) singularity/current   7) cmake/latest          9) gcc/9
  2) dot                   4) vgl/2.5.1/64          6) rhel7/global          8) python/3.8

Then I got this error:

[ir-inca1@login-e-10 moose]$ cd petsc/
[ir-inca1@login-e-10 petsc]$ make PETSC_DIR=/home/ir-inca1/projects/moose/scripts/../petsc PETSC_ARCH=arch-moose check
Running check examples to verify correct installation
Using PETSC_DIR=/home/ir-inca1/projects/moose/scripts/../petsc and PETSC_ARCH=arch-moose
Possible error running C/C++ src/snes/tutorials/ex19 with 1 MPI process
See http://www.mcs.anl.gov/petsc/documentation/faq.html
srun: error: Unable to allocate resources: Invalid account or account/partition combination specified
Possible error running C/C++ src/snes/tutorials/ex19 with 2 MPI processes
See http://www.mcs.anl.gov/petsc/documentation/faq.html
srun: error: Unable to allocate resources: Invalid account or account/partition combination specified
1,5c1
< lid velocity = 0.0016, prandtl # = 1., grashof # = 1.
<   0 SNES Function norm 0.0406612 
<   1 SNES Function norm 4.12227e-06 
<   2 SNES Function norm 6.098e-11 
< Number of SNES iterations = 2
---
> srun: error: Unable to allocate resources: Invalid account or account/partition combination specified
/home/ir-inca1/projects/moose/petsc/src/snes/tutorials
Possible problem with ex19 running with hypre, diffs above
=========================================
1,9c1
< lid velocity = 0.0625, prandtl # = 1., grashof # = 1.
<   0 SNES Function norm 0.239155 
<     0 KSP Residual norm 0.235858 
<     1 KSP Residual norm < 1.e-11
<   1 SNES Function norm 6.81968e-05 
<     0 KSP Residual norm 2.30906e-05 
<     1 KSP Residual norm < 1.e-11
<   2 SNES Function norm < 1.e-11
< Number of SNES iterations = 2
---
> srun: error: Unable to allocate resources: Invalid account or account/partition combination specified
/home/ir-inca1/projects/moose/petsc/src/snes/tutorials
Possible problem with ex19 running with mumps, diffs above
=========================================
1,2c1
< lid velocity = 0.0025, prandtl # = 1., grashof # = 1.
< Number of SNES iterations = 2
---
> srun: error: Unable to allocate resources: Invalid account or account/partition combination specified
/home/ir-inca1/projects/moose/petsc/src/snes/tutorials
Possible problem with ex19 running with superlu_dist, diffs above
=========================================
Completed test examples

**Comment by GiudGiud** [comment, marked as answer]
See https://github.com/idaholab/moose/discussions/16546, issues with srun
